src/components/CardsContainer/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { Grid, makeStyles, Theme } from '@material-ui/core';
+import { ReactNode } from 'react';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  grid: {
+    width: '100%',
+    marginBlockStart: '3rem',
```

Comment:
Why did you use `marginBlockStart` instead of `marginTop` ?

Reply:
I used it because it is the same thing as margin but it has a cool feature that depends on the layout model. [See](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-block-start)

Reply:
Ok, thanks 👍🏿